test/functionalTest/cases/706_direct_https_notifications/direct_https_notifications.test

```diff
@@ -0,0 +1,213 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Direct HTTPS notifications
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+accumulatorStart 127.0.0.1 1443 https
+
+--SHELL--
+
+payload='{
+    "entities": [
+        {
+            "type": "Room",
+            "isPattern": "false",
+            "id": "Room1"
+        }
+    ],
+    "attributes": [
+        "temperature"
+    ],
+    "reference": "https://127.0.0.1:1443/notify",
```

Comment:
Add a .test with `https://127.0.0.1/notify`, in order to see if CB is able to use the default HTTPS port in that case (443).

Of course, in that case accumulator needs to run in 443 instead of 1443.